dashboard/app/controllers/census_reviewers_controller.rb

```diff
@@ -0,0 +1,54 @@
+class CensusReviewersController < ApplicationController
+  before_action :authenticate_user!
+  before_action :require_census_reviewer_permissions
+
+  def require_census_reviewer_permissions
+    return if current_user.permission?(UserPermission::CENSUS_REVIEWER)
+    raise(CanCan::AccessDenied, 'user must be a census reviewer')
+  end
+
+  def create
+    begin
+      submission = Census::CensusSubmission.find(params[:census_submission_id])
+    rescue ActiveRecord::RecordNotFound
```

Comment:
Rather than raising and rescuing an error, find via `find_by(id: ...`) and check for nil?

Reply:
Done.